static/recommend.js

```diff
@@ -1,5 +1,5 @@
 // Replace 'YOUR_API_KEY' below with your API key retrieved from https://www.themoviedb.org
-var myAPI = 'YOUR_API_KEY'  // global string to be consistent with future usages elsewhere
+var myAPI = '6b51ba6ea86b6d2ff952da7819bdfffa'  // global string to be consistent with future usages elsewhere  // global string to be consistent with future usages elsewhere
```

Comment:
This can't be approved as the API key is meant to be confidential and there's a rate limit where you could hit only 50 requests per second ([Read more](https://developer.themoviedb.org/docs/rate-limiting)). Everyone might start using this API if it's exposed to the public.